x/compliance/keeper/keeper.go

```diff
@@ -93,6 +100,29 @@ func (k Keeper) GetAddressDetails(ctx sdk.Context, address sdk.Address) (*types.
 		return nil, err
 	}
 
+	// Filter verification details by issuer's existance
+	var newVerifications []*types.Verification
+	for _, verification := range addressDetails.Verifications {
+		issuerAddress, err := sdk.AccAddressFromBech32(verification.IssuerAddress)
+		if err != nil {
+			return nil, err
+		}
+		exists, err := k.IssuerExists(ctx, issuerAddress)
+		if err != nil {
+			return nil, err
+		}
+		if exists {
+			newVerifications = append(newVerifications, verification)
+		}
+	}
+	if len(newVerifications) != len(addressDetails.Verifications) {
```

Comment:
Also could you disable updating address details in getter function, since it also can break consensus

Reply:
Oops, I forgot removing same thing in another function.